openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -1,18 +1,24 @@
 # -*- coding: utf-8 -*-
 """Maya look extractor."""
+import logging
+from abc import ABCMeta, abstractmethod
+
```

Comment:
any reasoning behind the separation of imports and not having them alphabetically ordered?

Reply:
Not really - this particular one was just missed. I do tend to keep separate builtins versus additional dependencies and local imports. But as far as I know that's not really based on any style guide, just used to doing it that way.

I've refactored it a bit now.

Reply:
yeah same, I usually have three blocks on this order:
<standard libraries>
<third-party libraries>
<local libraries>